library/core/src/iter/traits/iterator.rs

```diff
@@ -4060,6 +4060,50 @@ pub trait Iterator {
     {
         unreachable!("Always specialized");
     }
+
+    /// Checks if the Iterator has a value.
+    /// 'has_items' is short-circuiting; in other words, it will stop processing
+    /// as soon as the function finds the item in the Iterator.
```

Comment:
```suggestion
    /// Returns `true` if the iterator contains a value.
    ///
    /// 'has_items' is short-circuiting; in other words, it will stop processing
    /// as soon as the function finds the item in the `Iterator`.
```

Made the summary line more consistent with `slice::contains`/`HashSet::contains`. Also remember that a line break is needed between the summary and the content (otherwise the whole thing gets treated as a summary line).